application/lib/view/widgets/product_card.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:application/model/product_model.dart';
+import 'package:application/utils/styles/styles.dart';
+import 'package:application/view_model/products_screen_view_model.dart';
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+class ProductCard extends StatelessWidget {
+  final ProductModel productModel;
+
+  const ProductCard({
+    Key? key,
+    required this.productModel,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final productsScreenviewModel =
```

Comment:
Itt is érvényes az , hogy ez a widget nem szabad hozzáférjen egy másik screen viewmodeljéhez. Vagy legyen itt simán meghívva a navigation függvény, vagy legyen saját viewmodelje.